flow/designs/nangate45/bp_be_top/config.mk

```diff
@@ -12,7 +22,10 @@ export ADDITIONAL_LIBS = $(sort $(wildcard ./designs/$(PLATFORM)/$(DESIGN_NAME)/
 
 # These values must be multiples of placement site
 # x=0.19 y=1.4
-export DIE_AREA    = 0 0 1550.02 1342.6
-export CORE_AREA   = 10.07 11.2 1540.14 1332.8
+export DIE_AREA    = 0 0 1000 800 
+export CORE_AREA   = 10.07 11.2 990 790 
```

Comment:
There's a comment right there that says it should be a multiple of the cell site.

Reply:
That comment gets copied along everywhere but I've really seen the need.  ifp will snap down the row to a site multiple.

Reply:
It's not great practice. Might as well just use a util % if there is no concern with being exact?

Reply:
I agree and think we should just use util %.

Reply:
util % is fine with square aspect ratio. Here the X and Y size of the blocks have been tuned specifically for macro based designs